index.html

```diff
@@ -141,12 +141,14 @@
 
       <h2>Introduction</h2>
 
-      <p>The Pointer Lock API provides for input methods of applications based on
-      the movement of the mouse, not just the absolute position of a cursor.  A
+      <p>The Pointer Lock API provides applications with the ability to directly interpret mouse movements
```

Comment:
"provide with" seems more appropriate in this case (see http://www.oxforddictionaries.com/definition/english/provide example "A submarine fleet is still essential in providing Canada with the necessary means to defend the world's longest coastline"). but perhaps this is a en-gb vs en-us thing?